src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -41,6 +41,8 @@ public static function register($mode = 0)
             return;
         }
 
+        $UtilPrefix = class_exists('PHPUnit_Util_ErrorHandler') ? 'PHPUnit_Util_' : 'PHPUnit\Util\\';
```

Comment:
Note that I used this upper-first convention in this PR for these compatibility variables that holds the class names to use